ui/apps/platform/src/Containers/Vulnerabilities/VirtualMachineCves/Overview/VirtualMachinesCvesTable.tsx

```diff
@@ -49,12 +52,22 @@ function VirtualMachinesCvesTable() {
                 renderer={({ data }) => (
                     <Tbody>
                         {data.map((virtualMachine) => {
+                            const virtualMachineSeverityCounts =
+                                countVirtualMachineSeverities(virtualMachine);
                             return (
                                 <Tr key={virtualMachine.id}>
                                     <Td dataLabel="Virtual machine">
                                         <Link to={''}>{virtualMachine.name}</Link>
                                     </Td>
-                                    <Td dataLabel="CVEs by severity">placeholder for ROX-30528</Td>
+                                    <Td dataLabel="CVEs by severity">
+                                        <SeverityCountLabels
+                                            criticalCount={virtualMachineSeverityCounts.CRITICAL}
+                                            importantCount={virtualMachineSeverityCounts.HIGH}
+                                            moderateCount={virtualMachineSeverityCounts.MEDIUM}
```

Comment:
Seeing `HIGH` and `MEDIUM` versus important and moderate, can you double-check what `vuln.cvssV3` has a value? Off the top of my head, I would have expected a number between 0 and 10. Elsewhere, I think we use a helper function for conversion to severity (when Red Hat severity is not available). If that sounds unfamiliar, David or Saif might be able to give second opinion.

Reply:
That's a good question. I just based it on the [vulnerability.proto file](https://github.com/stackrox/stackrox/blob/master/proto/api/v2/vulnerability.proto#L113C3-L119C18), but I could be using the wrong field.

Reply:
Yeah, it looks like from the `EmbeddedVulnerability` type:

```
 cvss: number;
```
and 
```
cvssV3: CVSSV3;
```
and `CVSSV3` is an object that can contain:

```
score: number;
severity: CVSSV3Severity;
```
but I also see in that same file there is another type:

```
type CVSSScore = {
    source: CVSSSource;
    url: string;
    cvssv3: CVSSV3;
};
```

So it is a bit confusing 🤔

Reply:
> Policy Violations use Critical, High, Medium, and Low. I'm not sure where these came from

Pre-acquisition. Saif might remember if both violations and vulnerabilities shared same terms, and then post-acquisition vulnerabilities moved to Red Hat terms.

Reply:
Delayed realization about **V3**. There are conventions for different CVSS scores V1, V2, V3 (and V4, in the future). I have seen recent problems because new vulnerability has **only** CVSS V4 and not V3.

That to say if response type is hard-coded to V3, without the same upward compatible property convention as elsewhere, then there will be a problem whenever scanner supports V4.